clang_tidy/clang_tidy.bzl

```diff
@@ -61,6 +61,9 @@ def _rule_sources(ctx):
     if hasattr(ctx.rule.attr, "srcs"):
         for src in ctx.rule.attr.srcs:
             srcs += [src for src in src.files.to_list() if src.is_source]
+    if hasattr(ctx.rule.attr, "hdrs"):
+        for hdr in ctx.rule.attr.hdrs:
+            srcs += [hdr for hdr in hdr.files.to_list() if (hdr.is_source and hdr.extension != "h")]
```

Comment:
What's wrong with the ".h" files?

Reply:
https://github.com/erenon/bazel_clang_tidy/blob/f52516cc533081804ad09ccad8580eeb22b3cb98/clang_tidy/clang_tidy.bzl#L117

It seems that target is cpp explicitly, so I exclude `h` file as c.

Reply:
Reverted to check all header files in `hdrs`

Reply:
https://github.com/erenon/bazel_clang_tidy/blob/c2fe98cfec0430e78bff4169e9ca0a43123e4c99/clang_tidy/clang_tidy.bzl#L114

It seems that target is `cpp` explicitly. So, I excluded `h` file as `c`. What do you think?

Reply:
That doesn't mean the .cpp extension, it means the C++ fragment https://bazel.build/rules/lib/cpp.